src/cmd/services/m3query/config/config.go

```diff
@@ -50,6 +50,8 @@ type Configuration struct {
 	// Metrics configuration.
 	Metrics instrument.MetricsConfiguration `yaml:"metrics"`
 
+	Tracing instrument.TracingConfiguration `yaml:"tracing"`
```

Comment:
Just curious of what happens if this section is not added to the config; does it just turn tracing off?